packages/core/src/util/tokenTransferInspector.ts

```diff
@@ -150,9 +161,20 @@ const toTokenTransferValue = async (
 
 /** Inspect a transaction and return a map of addresses and their balances. */
 export const tokenTransferInspector: TokenTransferInspector =
```

Comment:
I've just noticed that this PR only added timeout for input resolvers, but forgot to handle failures and timeouts of AssetProvider (e.g. https://github.com/input-output-hk/cardano-js-sdk/blob/b3f3280c9ddc0b9c0122c9c4be0eacc2201a7faf/packages/core/src/util/tokenTransferInspector.ts#L142 which can also fail/timeout) which is in scope of LW-10504

Reply:
Should be fixed by 6b56dce807feb76cff36b051a10c36da0d584d4c